src/Spa/Generated/Pages.elm

```diff
@@ -473,23 +473,23 @@ bundle bigModel =
 
 
 view : Model -> Document Msg
-view =
-    bundle >> .view
+view model =
```

Comment:
How did this file change? Did you edit it or did you regenerate it somehow?

Reply:
I did not edit it, I think it got regenerated when I ran the code.